Lessons/Lesson 03/InClassActivity.playground/section-1.swift

```diff
@@ -25,3 +25,64 @@ import Foundation
 // Bonus TO DO: Write a program that prints the numbers from 1 to 100. But for multiples of three print “Fizz” instead of the number and for the multiples of five print “Buzz”. For numbers which are multiples of both three and five print “FizzBuzz”.
 
 // BONUS TODO: The first fibonacci number is 0, the second is 1, the third is 1, the fourth is two, the fifth is 3, the sixth is 5, etc. The Xth fibonacci number is the sum of the X-1th fibonacci number and the X-2th fibonacci number. Print the 37th fibonacci number below
+
+var age:Int?
+var name:String?
+
+println("Hello \(name), you are \(age)")
+println("Hello \(name), your name is countElements(name) characters long")
+
+if age >= 21 {
+    println("You can drive, vote, and drink, but not at the same time")
+} else if age > 18 {
+    println("You can vote")
+} else if age > 16 {
+    println("You can drive")
+}
+
+for var i = 0; i < 50 ; i++ {
+    print(i * 7 - 1)
+}
+
+var number:Int?
```

Comment:
this is a legal variable definition using the ? optional operand, but it will cause your comparison expression below to throw an error if you do not set number to a value. This is because it is still set to nil and running modulo on a nil value will throw exceptions.